Formula/sstp-client.rb

```diff
@@ -1,8 +1,8 @@
 class SstpClient < Formula
   desc "SSTP (Microsofts Remote Access Solution for PPP over SSL) client"
   homepage "https://sstp-client.sourceforge.io/"
-  url "https://downloads.sourceforge.net/project/sstp-client/sstp-client/sstp-client-1.0.13.tar.gz"
-  sha256 "961258fca0795d8ad60b047942cf7cb53d025d353fd1e4ba08c2b75799f5321b"
+  url "https://gitlab.com/eivnaes/sstp-client/-/archive/1.0.15/sstp-client-1.0.15.tar.bz2"
```

Comment:
The build failed because you switched to a GitLab source tarball, which no longer contains a generated `configure` script. The Sourceforge tarball will work if you switch to that one.

Reply:
Thanks for the comment! I'm in contact with the maintainer of sstp-client to clarify which plattform is preferred. Currently the SourceForge links seems to be the easiest way for the update, but in the future GitLab is going to be used according to a statement in the forums: https://sourceforge.net/p/sstp-client/discussion/1499218/thread/d485651bda/#a7ab

Reply:
We can always use the Sourceforge tarball for now. There's one for 1.0.15.